salon_creator/lib/screens/lesson_creation_screen.dart

```diff
@@ -21,32 +21,44 @@ class LessonCreationScreen extends StatefulWidget {
 }
 
 class _LessonCreationScreenState extends State<LessonCreationScreen> {
-  bool _publish = false;
-  bool isFormFilled = false;
-  bool operationInProgress = false;
-  List<File> resources = [];
-  List<String> resourcesList = [];
-  TextEditingController categoriesController = TextEditingController();
-  TextEditingController descriptionController = TextEditingController();
-  TextEditingController lessonNameController = TextEditingController();
+  bool _isPublish = false;
+  bool _isFormFilled = false;
+  bool _operationInProgress = false;
+  List<File> _resources = [];
+  List<String> _resourcesList = [];
+  TextEditingController _categoriesController = TextEditingController();
+  TextEditingController _descriptionController = TextEditingController();
+  TextEditingController _lessonNameController = TextEditingController();
   XFile _imageFile;
   XFile _mediaFile;
-  XFile thumbnailFile;
-  XFile image;
+  XFile _thumbnailFile;
+
+  Future<XFile> imagePicker() async {
```

Comment:
Similar to the pickFile function, this could be better named:

```suggestion
  Future<XFile> _pickImage() async {
```